gems/quilt_rails/app/views/quilt/ui/react_render_error.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>React Render Error</title>
+  <meta name="viewport" content="width=device-width,initial-scale=1">
+  <meta http-equiv="refresh" content="3;URL='/'" />
```

Comment:
If we add 
```
<link
  rel="stylesheet"
  href="https://unpkg.com/@shopify/polaris@5.1.0/dist/styles.css"
/>
```

we can use Polaris CSS components and make the page at least a little bit nice :)

Reply:
Stylesheet link added, but the page still needs some cosmetic changes. This can be done later though, I'll add a changelog for this and then merge once everything is 🟢